nemo/collections/asr/data/audio_to_text.py

```diff
@@ -75,7 +75,9 @@ def _speech_collate_fn(batch, pad_id):
     has_audio = audio_lengths[0] is not None
     if has_audio:
         max_audio_len = max(audio_lengths).item()
-    max_tokens_len = max(tokens_lengths).item()
+    has_tokens = tokens_lengths[0] is not None
```

Comment:
what's the motivation for this change?

Reply:
yes, it now supports in transcribe() tensor dataset which may return None for token/lens instead of dummy tensor